.github/workflows/test.yml

```
@@ -52,7 +57,7 @@ jobs:
       - uses: zendesk/checkout@v2
```

While you’re at it, let’s make those warnings go away.
```suggestion
      - uses: zendesk/checkout@v3
```